conf/routes

```diff
@@ -57,6 +57,8 @@ GET     /adminapi/label/:labelId                             @controllers.AdminC
 GET     /adminapi/labelCounts                                @controllers.AdminController.getAllUserLabelCounts
 GET     /adminapi/attributes/all                             @controllers.AdminController.getAllAttributes
 
```

Comment:
No need to have a newline separating the validationCounts endpoint from the rest above it.